turbopack/crates/turbo-tasks/src/manager.rs

```diff
@@ -1776,6 +1778,27 @@ pub async fn spawn_blocking<T: Send + 'static>(func: impl FnOnce() -> T + Send +
     result
 }
 
+pub async fn spawn_blocking_rayon<T: Send + 'static>(
```

Comment:
```suggestion
pub async fn spawn_blocking_compute<T: Send + 'static>(
```

let's assume rayon and tokio are implementation details and avoid naming these functions by library name and name them by intent instead.